include/mbedtls/cipher.h

```diff
@@ -473,7 +476,8 @@ static inline int mbedtls_cipher_get_key_bitlen( const mbedtls_cipher_context_t
  * \param ctx      The context of the cipher. Must be initialized.
  *
  * \return         The type of operation: #MBEDTLS_ENCRYPT or
- *                 #MBEDTLS_DECRYPT, or #MBEDTLS_OPERATION_NONE if \p ctx
+ *                 #MBEDTLS_DECRYPT.
```

Comment:
Style: Don't put `MBEDTLS_DECRYPT` in a separate line.

Reply:
ok